site/sfguides/src/hex_titanic/titanic.md

```diff
@@ -0,0 +1,192 @@
+author: aefendic@hex.tech and chase.romano@snowflake.com
+id: hex
```

Comment:
id should be a combination of "End to End Model Development with Hex and Snowpark" with lowercase and "_". for example, "getting_started_with_snowpark_python"

Reply:
updated, thanks for flagging!